src/cpu-template-helper/src/dump/host.rs

```diff
@@ -20,9 +29,73 @@ pub struct HostFingerprint {
 }
 
 pub fn dump() -> Result<HostFingerprint, Error> {
-    // TODO: Add implementation to dump host fingerprint.
+    let commands = vec![
+        "uname -r",
+        "dmidecode -t bios",
```

Comment:
You may find the info you need in `/sys/devices/virtual/dmi/id/bios_release` without having to shell out.

Reply:
Thanks! This is what I want!

Reply:
Unfortunately the path is available since kernel 5.8.
https://github.com/torvalds/linux/commit/f5152f4ded3ce6d332d5e4f9d7e325c3b81cae1b

Reply:
Having offline discussion with Pablo, we decided to use dmidecode command at the moment and put TODO there to consume sysfs after kernel 4.14 deprecation.